modules/core/src/test/scala/org/tessellation/infrastructure/trust/TrustInfoGenerator.scala

```diff
@@ -0,0 +1,32 @@
+package org.tessellation.infrastructure.trust
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.TrustInfo
+
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Gen
+
+object TrustInfoGenerator {
+
+  val genDouble: Gen[Double] = arbitrary[Double]
+
+  val genPeerId: Gen[PeerId] = arbitrary[PeerId]
```

Comment:
Can you point me to one that does?

Reply:
I think we don't have one, just because of that. You'd have to derive the peer identifiers from public keys. I think @kriscieslak  already did that somewhere in tests.

Reply:
@kriscieslak, can you point me to where?